.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 default_language_version:
-  python: python3.9
+  python: python3.8
```

Comment:
It seems pre-commit doesn't like 3.8 much in the CI.

Definitely agree that we should use the minimum supported version in the dev environments